client/python/samples/resnet_grpc_predict.py

```diff
@@ -68,4 +68,4 @@
 
     # response post processing
     label, confidence_score = resnet_postprocess(response, output_name)
-    print(f"Image {i} has been classified as {label} with {confidence_score}% confidence")
+    print(f"Image {i+1} has been classified as {imagenet_classes[label]} with {confidence_score}% confidence")
```

Comment:
why not return `imagenet_classes[label]` from `resnet_postprocess` instead of raw label?